Todo CLI/todoCLI.js

```diff
@@ -0,0 +1,65 @@
+const readline = require("readline");
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+console.log('  Welcome to Todo CLI! \n ------------------------ ');  
+
+function menu (){
+    const option = `(v) View • (n) New • (cX) Complete • (dX) Delete • (q) Quit \n `
+    rl.question(option, answer => {
+        if(answer === 'v') return view();
+        else if(answer === 'n') return add();
+        else if(answer.includes('c')) return complete(answer);
+        else if(answer[0] === 'd') return del(answer);
+        else if(answer === 'q') return quit();
+        else return console.log(`no option for that :(`);
+    });
+};
+
+let checkBox = `[ ]`;
+let lists = [];
+
+function view () {
+    if(lists.length <= 0) console.log('list is empty....');
+    else{
+        lists.forEach((list, index) => {
+            console.log(`${index} ${list}\n`);
+        });
+    }
+    return menu();
+}
+
+function add () {
+    rl.question(`what?\n`, answer => {
+        let list = `${checkBox}${answer}`
+        lists.push(list);
+        return menu();
+    });
+}
+    
+function complete (answer) {
+    //string to number
+    let num = parseInt(answer.substr(1));
+    //check the box
+    lists[num] = lists[num].replace("[ ]", "[✓]")
```

Comment:
We can add a conditional in here and also in the `del` function to check if `lists[num]` exists before calling methods on it to prevent `TypeError`. If we try to complete or delete a task that doesn’t exist, the program will crash.